stopcovid/drills/drill_content/drills.json

```diff
@@ -12,7 +13,14 @@
           }
         ],
         "response_user_profile_key": "language"
-      },
+      }
+    ]
+  },
+  "01-intake": {
+    "name": "Intake",
+    "slug": "01-intake",
+    "auto_continue": true,
+    "prompts": [
```

Comment:
If you're making a similar change to drills in admin, I was wondering if this is going to make it difficult to compile stats later. i.e. should drill slugs be immutable?

Reply:
Yeah... im wondering if we should name this drill `00-language` to not screw with reporting. Good thought.